src/Microsoft.AspNetCore.Routing/Matching/CandidateSet.cs

```diff
@@ -18,8 +23,13 @@ public sealed class CandidateSet
 
         private CandidateState[] _additionalCandidates;
 
-        // Provided to make testing possible/easy for someone implementing
-        // an EndpointSelector.
+        /// <summary>
+        /// Creates a new candidate set with the provided list of endpoints and associated scores.
```

Comment:
nit: Initializes a new instance of the `CandidateSet` structure with the provided list of endpoints and associated scores.